src/CLI/OutputGenerator.cs

```diff
@@ -47,14 +48,14 @@ public void WriteBanner(IOptions options)
         public void WriteThirdPartyNoticeOutput(string pathToFile)
         {
             WriteAppBanner();
-            _writer.WriteLine("Opening Third Party Notices in default browser. If this fails, manually open \"{0}\"", pathToFile);
+            _writer.WriteLine(DisplayStrings.ThirdPartyNoticeFormat, pathToFile);
```

Comment:
The writer will already use CurrentCulture under the hood (it's Console.Out in production code)